templates/typescript/licenseInfo.mustache

```diff
@@ -0,0 +1,27 @@
+/*
+ *                       ######
+ *                       ######
+ * ############    ####( ######  #####. ######  ############   ############
+ * #############  #####( ######  #####. ######  #############  #############
+ *        ######  #####( ######  #####. ######  #####  ######  #####  ######
+ * ###### ######  #####( ######  #####. ######  #####  #####   #####  ######
+ * ###### ######  #####( ######  #####. ######  #####          #####  ######
+ * #############  #############  #############  #############  #####  ######
+ *  ############   ############  #############   ############  #####  ######
+ *                                      ######
+ *                               #############
+ *                               ############
+ * Adyen NodeJS API Library
+ * Copyright (c) 2022 Adyen B.V.
```

Comment:
I would be nice if we could make this dynamic: 
```suggestion
 * Copyright (c) {{year}} Adyen B.V.
```

Reply:
Good suggestion, was not aware this was possible!

Reply:
Bummer, maybe we can leave it out then?
```
Copyright (c) Adyen N.V.
```